modules/logging-architecture-overview.adoc

```diff
@@ -6,24 +6,30 @@
-
-* `Visualization` - Graphical interface for searching, querying, and viewing stored logs.
-
-These components are managed by Operators and Custom Resource (CR) YAML files.
+The major components of the {logging} are:
+
+Collector:: The collector is a daemonset that deploys pods to each {product-title} node. It collects log data from each node, transforms the data, and forwards it to configured outputs. You can use the Vector collector or the legacy Fluentd collector.
++
+--
+include::snippets/logging-fluentd-dep-snip.adoc[]
+--
+
+Log store:: The log store stores log data for analysis and is the default output for the log forwarder. You can use the default LokiStack log store, the legacy Elasticsearch log store, or forward logs to additional external log stores.
++
+--
+include::snippets/logging-elastic-dep-snip.adoc[]
+--
+
+Visualization:: You can use a UI component to view a visual representation of your log data. The UI provides a graphical interface to search, query, and view stored logs. The {product-title} web console UI is provided by enabling the {product-title} console plugin, but you can also use Kibana.
++
+--
+include::snippets/logging-kibana-dep-snip.adoc[]
+--
 
 The {logging-title} collects container logs and node logs. These are categorized into types:
 
-* `application` - Container logs generated by user applications running in the cluster, except infrastructure container applications.
-
-* `infrastructure` - Logs generated by infrastructure components running in the cluster and {product-title} nodes, such as journal logs. Infrastructure components are pods that run in the `openshift*`, `kube*`, or `default` projects.
-
-* `audit` - Logs generated by auditd, the node audit system, which are stored in the */var/log/audit/audit.log* file, and logs from the `auditd`, `kube-apiserver`, `openshift-apiserver` projects, as well as the `ovn` project if enabled.
+Application logs:: Container logs generated by user applications running in the cluster, except infrastructure container applications.
 
-The logging collector is a daemonset that deploys pods to each {product-title} node. System and infrastructure logs are generated by journald log messages from the operating system, the container runtime, and {product-title}.
+Infrastructure logs:: Container logs generated by infrastructure namespaces: `openshift*`, `kube*`, or `default`, as well as journald messages from nodes.
 
-Container logs are generated by containers running in pods running on the cluster. Each container generates a separate log stream. The collector collects the logs from these sources and forwards them internally or externally as configured in the `ClusterLogForwarder` custom resource.
+Audit logs:: Logs generated by auditd, the node audit system, which are stored in the */var/log/audit/audit.log* file, and logs from the `auditd`, `kube-apiserver`, `openshift-apiserver` projects, as well as the `ovn` project if enabled.
```

Comment:
and audit logs from the `auditd`, `kube-apiserver`, `openshift-apiserver` services. (It is better to replace projects with services.)